addons/point_of_sale/static/tests/tours/ReceiptScreen.tour.js

```diff
@@ -70,6 +70,17 @@ registry.category("web_tour.tours").add("ReceiptScreenTour", {
             PaymentScreen.clickPaymentMethod("Bank"),
             PaymentScreen.clickValidate(),
             Order.hasLine({ customerNote: "Test customer note" }),
+            ReceiptScreen.clickNextOrder(),
+
+            //Test shipping date in receipt
+            ProductScreen.addOrderline("Letter Tray", "10", "5"),
```

Comment:
Instead of creating a new order could you make the first one to use ship later ? This way we avoid adding useless steps to the tour